lib/urldata.h

```diff
@@ -1104,6 +1104,7 @@ struct connectdata {
  */
 struct PureInfo {
   int httpcode;  /* Recent HTTP, FTP, RTSP or SMTP response code */
+  char *httpresponse; /* Recent HTTP, FTP, RTSP or SMTP response string */
```

Comment:
I think using `http` in the name is unfortunate when it is valid for many more protocols. I realize 'httpcode' is named like that already, but I consider that to be subject for a later fix.